test/integration/test_session.py

```diff
@@ -83,7 +83,8 @@ def test_session_from_id(self, service):
         isa_circuit = pm.run([bell()])
         with Session(backend=backend) as session:
             sampler = SamplerV2(mode=session)
-            sampler.run(isa_circuit)
+            job = sampler.run(isa_circuit)
+            job.result()
```

Comment:
Might be missing function, but where is `session.details()` called in this test? is it called in `mock_create_session.assert_not_called()`?

Reply:
Sorry my comment was lacking context - details() is called in from_id() - https://github.com/kt474/qiskit-ibm-runtime/blob/main/qiskit_ibm_runtime/session.py#L337